courseraresearchexports/models/ExportRequest.py

```diff
@@ -216,13 +216,13 @@ def scope_id(self):
     @property
     def scope_name(self):
         """
-        Human readable name for this scope context. course slugs for courses,
-        partner short names for partners, but only group ids for groups (api is
-        not open)
+        Human readable name for this scope context. Partner short names for
+        partners, but only group ids for groups and course ids for courses(apis
+        are not open)
         :return:
         """
         if self._course_id:
-            return utils.lookup_course_slug_by_id(self._course_id)
+            return self._course_id
```

Comment:
The scopeName is used to give human readable name to the docker container. The 
Currently `lookup_course_slug_by_id` is available only for public course_ids, hence it fails for private courses. 

Will try to add try, catch block to first try resolving slug, otherwise return course_id.

Reply:
Let's add the try-catch since it will be nice to have human readable strings in most cases :)